CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This file is used to list changes made in each version of the docker cookbook.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 7.3.0 (unreleased)
```

Comment:
We need this for our automated release system.
```suggestion
## Unreleased
```

Reply:
Noticed that in the checks! should be good now